pages/roadmap/[...slug].tsx

```diff
@@ -139,34 +141,35 @@ export default function RoadmapPage(props: InferGetServerSidePropsType<typeof ge
       setIsPendingChildrenLoading(false);
     };
     fetchPendingChildren();
-  }, [issue_number, repo, owner, isRootIssueLoading, pendingChildrenState.value]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [issue_number, repo, owner, isRootIssueLoading, pendingChildrenState.length]);
```

Comment:
passing `@hookstate/core`'s `State<>` objects (via `.value`, `.get()`, or even directly) as useEffect dependencies will cause useEffect to constantly execute. PSA @whizzzkid @djmcquillan .